resources/views/layouts/base.blade.php

```diff
@@ -37,9 +37,14 @@
     <link href="/css/base.css" rel="stylesheet">
     <link href="/css/toastr.min.css" rel="stylesheet">
     <link href="/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/morris.js/0.5.1/morris.css">
 
     <link rel="shortcut icon" href="favicon.ico">
     @yield('css')
+
+    <script src="/js/jquery-1.11.3.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>
```

Comment:
This assets could be required using npm, and later pre-compiled using elixir/gulp
But probably we should leave this for later, this PR is already quite big.